userdocs/userdocs.docc/userdocs.md

```diff
@@ -0,0 +1,45 @@
+# vscode-swift
+
+@Metadata {
+   @TechnologyRoot
+}
+
+Language support for Swift in Visual Studio Code.
+
+@Metadata {
+    @DisplayName("vscode-swift")
```

Comment:
this might be better merged in under line 4, to keep the metadata directive elements grouped together.